client/klay_client.go

```diff
@@ -237,6 +237,18 @@ func (ec *Client) TransactionReceipt(ctx context.Context, txHash common.Hash) (*
 	return r, err
 }
 
+// TransactionReceiptRpcOutput returns the receipt of a transaction by transaction hash as a rpc output.
+func (ec *Client) TransactionReceiptRpcOutput(ctx context.Context, txHash common.Hash) (map[string]interface{}, error) {
```

Comment:
how about this?
`func (ec *Client) TransactionReceiptRpcOutput(ctx context.Context, txHash common.Hash) (r map[string]interface{}, error) {`
~var r map[string]interface{}~